x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/index.test.ts

```diff
@@ -228,6 +228,14 @@ describe('policy details: ', () => {
               policy: {
                 value: {
                   windows: {
+                    advanced: {
```

Comment:
I'm wondering if the entire `advanced` section should be special in that we don't actually add it to the defaults.  Instead, we wouldn't initialize any of the fields and not send them down to the Endpoint unless the user specifically adds them.  

It would be an extra step in adding new fields, but it might be easier to just include them for TS, etc.

What are your thoughts @parkiino @paul-tavares

Reply:
23238e9d29504f34a9e774cad99443b830951c52